internal/callbacker/callbacker_api/callbacker_api.proto

```diff
@@ -0,0 +1,55 @@
+syntax = "proto3";
+
+option go_package = ".;callbacker_api";
+
+package callbacker_api;
+
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/empty.proto";
+
+service CallbackerAPI{
+  rpc Health (google.protobuf.Empty) returns (HealthResponse) {}
+  rpc SendCallback (SendCallbackRequest) returns (google.protobuf.Empty) {}
+}
+
+// Note: Values of the statuses have a difference between them in case
+// it's necessary to add another values in between. This will allow to
+// create new statuses without changing the values in the existing db.
+enum Status {
```

Comment:
Could we use here just an integer instead?

I think `callbacker` probably doesn't need to know any logic about statuses anyway

Reply:
We have prometeus stats based on the status
![image](https://github.com/user-attachments/assets/879b6fd8-6226-44a0-8068-337f583f4974)

Reply:
I see now, it's used here: https://github.com/bitcoin-sv/arc/blob/93d51c633c3983b586c392391355bc18cf28dd5e/internal/callbacker/callbacker.go#L179

Nevermined